autoscheduler/frontend/src/components/LandingPage/SelectTerm/SelectTerm.tsx

```diff
@@ -14,19 +16,43 @@ const options = [
   'Semester 6',
 ];
 
+// Maps between the term description and its actual value (which we'll use to store)
+// We might not need this in the future?
+const termMap: { [key: string]: any } = {
+  // This lint is temporary, and shouldn't be needed for the future
+  // eslint-disable-next-line quote-props
+  'None': -1,
+  'Semester 1': 201931,
+  'Semester 2': 201931,
+  'Semester 3': 201931,
+  'Semester 4': 201931,
+  'Semester 5': 201931,
+  'Semester 6': 201931,
+};
+
 const SelectTerm: React.SFC = () => {
   // anchorEl tells the popover menu where to center itself. Null means the menu is hidden
   const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
   const open = Boolean(anchorEl);
-  const [selectedTerm, selectTerm] = React.useState(options[0]);
+  const [selectedTerm, setSelectedTerm] = React.useState(options[0]);
+
+  const dispatch = useDispatch();
 
   const handleClick = (event: React.MouseEvent<HTMLElement>): void => {
     setAnchorEl(event.currentTarget);
   };
 
   const handleClose = (option: string): void => {
     setAnchorEl(null);
-    selectTerm(option);
+    setSelectedTerm(option);
+
+    // Get the corresponding option given the term's description
+    const term: number = termMap[option];
+
+    dispatch(setTerm({ term }));
+
+    // Redirect to the main page
+    window.location.assign('/schedule');
```

Comment:
This works, but I think it owuld be better to use `navigate` from the [Reach router library](https://reach.tech/router/api/navigate), since that's our standard navigation library